modulefiles/module_gwci.orion.lua

```diff
@@ -7,7 +7,7 @@ prepend_path("MODULEPATH", "/apps/contrib/NCEP/hpc-stack/libs/hpc-stack/modulefi
 load(pathJoin("hpc", "1.1.0"))
 load(pathJoin("hpc-intel", "2018.4"))
 load(pathJoin("hpc-impi", "2018.4"))
-load(pathJoin("netcdf","4.7.4"))
+load(pathJoin("netcdf","4.7.2-parallel"))
```

Comment:
Why not 4.7.4 parallel?

Also, check with NCEPLIBS, but I thought when impi is loaded, the netcdf version already supports parallel.

Reply:
After talking with Terry, nccmp is the issue (it has incorrect prerequisites on Orion). Moving to `nccmp/1.8.7.0` should fix things, as that module has the correct prerequisites. Hera should also be updated for consistency.

Reply:
Walter figured out that nccmp 1.8.7.0 would link ok with intel MPI